Doc/library/functions.rst

```diff
@@ -668,16 +668,15 @@ are always available.  They are listed here in alphabetical order.
       sign: "+" | "-"
       infinity: "Infinity" | "inf"
       nan: "nan"
-      digitpart: `!digit` (["_"] `!digit`)*
+      digit: <a Unicode decimal digit, i.e. all characters the Unicode general category Nd>
```

Comment:
Hmm, maybe should be shortened a bit?
```suggestion
      digit: <a decimal digit, i.e. characters in the Unicode general category Nd>
```

Reply:
I took the grammar fix you suggested, but left the Unicode in. I think it's an important part of `digit` that's also easy to forget (e.g. https://github.com/python/cpython/issues/114070#issuecomment-1891424747) so doesn't hurt to emphasise. I'm not too worried about brevity here.